src/jit/codegenlegacy.cpp

```diff
@@ -18830,15 +18830,17 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
                                                    (ssize_t)stubAddr);
                             // The stub will write-back to this register, so don't track it
                             regTracker.rsTrackRegTrash(compiler->virtualStubParamInfo->GetReg());
-                            getEmitter()->emitIns_R_R_I(INS_ldr, EA_PTRSIZE, REG_JUMP_THUNK_PARAM,
```

Comment:
Yes, if you want to merge this, please alter it such that there is no change to the code for the non-CoreRT `LEGACY_BACKEND` path.